hub.py

```diff
@@ -328,6 +328,12 @@ def apply_hub_template_fixes(self, generated_config, proxy_secret_key):
         them in this function.
         """
         hub_template = self.spec['template']
+        hub_name = self.spec['name']
+
+        if hub_name == "mills":
```

Comment:
In general we should try to avoid special casing any particular hub in our deployment script. I'd like for us to remove the special casing for hub-templates too, but some other time.

One suggestion here is to allow `domain` to be a list, rather than just a string as it is now. This should cause minimal issues, no hard coding, and pass through all the things we want.

Reply:
Making `domain` a list, is a great idea <3 Thank you!

I have one curiosity though, if we were to add the `datahub.mills.edu` ingress part into `hubs.yaml` would that have merged ok with the one in `generated_config`? (I understand that it's better to have similar config in the same place, I'm just wondering how smart the config merging process is)

Reply:
I think right now lists don't get merged, since there isn't unfortunately an easy way to merge them :( If we can figure that out, that'd be great! Not sure of an obvious solution though.